build_artifacts/v1/v1.0/v1.0.0/gpu.env.in

```diff
@@ -17,7 +17,7 @@ conda-forge::py-xgboost-gpu[version='>=1.7.6']
 conda-forge::jupyterlab[version='>=3.6.5']
 conda-forge::ipywidgets[version='>=8.1.0']
 conda-forge::conda[version='>=23.7.3']
-conda-forge::boto3[version='>=1.28.35']
+conda-forge::boto3[version='>=1.28.35,<1.28.65']
```

Comment:
Will need identical changes to this file as compared with CPU. The `botocore` constraint is missing here

Reply:
Thanks for the catch - fixed it for GPU